website/source/docs/providers/azurerm/r/virtual_network.html.markdown

```diff
@@ -57,7 +57,7 @@ The following arguments are supported:
 * `dns_servers` - (Optional) List of names of DNS servers previously registered
     on Azure.
 
-* `subnet` - (Required) Can be specified multiple times to define multiple
+* `subnet` - (Optional) Can be specified multiple times to define multiple
```

Comment:
@jen20 this is exactly the same behaviour as for Network Security Group / Network Security Rule

From Slack (with @phinze):

```

stack72 [2:01 PM] 
oh they can’t be mixed?

phinze [2:01 PM] 
right because they'll always fight

stack72 [2:01 PM] 
ah so you would set up a basic network_acl and ​*then add*​ the rules?

​[2:01] 
now i see

​[2:01] 
that makes more sense

phinze [2:01 PM] 
yep, all nested or all _rule is the key
```